low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy.sh

```diff
@@ -273,9 +273,9 @@ fi
     # below should be removed and setup_yum_repo should be able to give cortx-py-utils.
     pkg_name="cortx-py-utils"
     build_url="http://cortx-storage.colo.seagate.com/releases/cortx/components/github/main/centos-7.8.2003/dev/cortx-utils/last_successful/"
-    yum install -y gcc rpm-build python36 python36-pip python36-devel python36-setuptools openssl-devel libffi-devel
+    yum install -y gcc rpm-build python36-devel openssl-devel libffi-devel
     yum install -y $build_url/$(curl -s $build_url/|grep $pkg_name|sed 's/<\/*[^"]*"//g'|cut -d"\"" -f1) ||:
-    python3 -m pip install toml pyyaml idna==2.8 
+    python3 -m pip install toml pyyaml
```

Comment:
Need toml also? Where we are using toml?

Reply:
I am not using it in my patch, but this seems cortx-py-utils pkg dependancy. 
while testing "sspl self provionser patch", we were getting error for this pkg, thats why mariyappan added this pkg.
@mariyappanp  can you please confirm.

I will test my patch without using these pkg on fresh vm.